exec/csnex/Exec.cs

```diff
@@ -7,6 +7,27 @@ namespace csnex
 {
     public class Executor
     {
+        sealed class ActivationFrame
+        {
+            public List<Cell> Locals;
+            public readonly int FrameSize;
```

Comment:
Do you need to keep track of this separately? Isn't this exactly the same as the size of `Locals`?

Reply:
Removed FrameSize member.